object-storage-oracle-cloud/src/main/java/io/micronaut/objectstorage/oraclecloud/OracleCloudStorageEntry.java

```diff
@@ -54,4 +57,14 @@ public InputStream getInputStream() {
     public GetObjectResponse getNativeEntry() {
         return objectResponse;
     }
+
+    @NonNull
+    @Override
+    public Map<String, String> getMetadata() {
+        if (CollectionUtils.isNotEmpty(objectResponse.getOpcMeta())) {
```

Comment:
maybe replace:

```java
        if (CollectionUtils.isNotEmpty(objectResponse.getOpcMeta())) {
             return objectResponse.getOpcMeta();
         } else {
             return Collections.emptyMap();
         }
```

with: 

```java
return CollectionUtils.isNotEmpty(objectResponse.getOpcMeta()) ? objectResponse.getOpcMeta() : Collections.emptyMap();
```

Reply:
A one-liner doesn't seem clearer to me in this context